tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -177,7 +178,23 @@ static void ReformatApiCatalog()
         static void RewriteReadme(string root, List<ApiMetadata> apis)
         {
             var readmePath = Path.Combine(root, "README.md");
-            var existing = File.ReadAllLines(readmePath).ToList();
+            RewriteApiTable(readmePath, apis, api => $"https://googleapis.dev/dotnet/{api.Id}/latest");
```

Comment:
This is what I was suggesting (or similar, not sure if api.Version is exactly what needs to go there) while the TOC issue is being fixed. (And in the README at the root of the repo as well).
```suggestion
            RewriteApiTable(readmePath, apis, api => $"https://googleapis.dev/dotnet/{api.Id}/{api.Version}");
```

Reply:
This line *only* affects the README at the root repo. The one in RewriteDocsRootIndex is only a relative URL to GitHub pages.
I'd like to make the README change separately, after we've done a tiny bit more investigation into the TOC issue.